examples/tree_1d_dgsem/elixir_advection_diffusion.jl

```diff
@@ -0,0 +1,87 @@
+
+using OrdinaryDiffEq
+using Trixi
+
+###############################################################################
+# semidiscretization of the linear advection equation
+
+advection_velocity = 1.0e-1
+equations = LinearScalarAdvectionEquation1D(advection_velocity)
+diffusivity() = 1.0e-1
+equations_parabolic = LaplaceDiffusion1D(diffusivity(), equations)
```

Comment:
```suggestion
advection_velocity = 0.1
equations = LinearScalarAdvectionEquation1D(advection_velocity)
diffusivity() = 0.1
equations_parabolic = LaplaceDiffusion1D(diffusivity(), equations)
```
Maybe for easier readability?